src/bundle.js

```diff
@@ -0,0 +1,46 @@
+import Mfs from 'memory-fs'
+import webpack from 'webpack'
```

Comment:
Looking around on github at nexe projects (and in my own experience). I've seen a lot of users use webpack on their own before building with nexe. Every app's bundling needs will be different and webpack is a uniquely flexible tool to make it fit together. I do think it would be good to make this pluggable though.